cime_config/testdefs/testmods_dirs/clm/Fates/shell_commands

```diff
@@ -1,2 +1,16 @@
 ./xmlchange CLM_BLDNML_OPTS="-no-megan" --append
 ./xmlchange BFBFLAG="TRUE"
+
+CASE=`./xmlquery CASE --value`
+SRCROOT=`./xmlquery SRCROOT --value`
+
+#CIME_OUTPUT_ROOT=`./xmlquery CIME_OUTPUT_ROOT --value`
+# CASEDIR=`./xmlquery CASEROOT --value`
+# env_test.xml IS NOT AVAILABLE TESTROOT=`./xmlquery TESTROOT --value`
+# env_test.xml IS NOT AVAILABLE TESTID=`./xmlquery TESTID --value`
+# env_test.xml IS NOT AVAILABLE TESTARGV=`./xmlquery TEST_ARGV --value`
+
+FATESDIR=$SRCROOT/src/fates
+FATESPARAMFILE=${SRCROOT}/src/fates/parameter_files/binaries/${CASE}-params.nc
+
+ncgen -o $FATESPARAMFILE $FATESDIR/parameter_files/fates_params_default.cdl
```

Comment:
Since, this builds the FATES param files from the CDL it should only change answers if the CTSM FATES param file isn't identical right? And since you didn't update FATES nor the default CTSM FATES param file in namelist_defaults, I take it answers are identical here?

Reply:
This has to be true, so resolving.